feature-libs/organization/account-summary/root/model/augmented.model.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-FileCopyrightText: 2022 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import '@spartacus/organization/administration/components';
```

Comment:
This import make `administration/components` and `administration/core` eager loaded. If you check `eature-libs_organization_account-summary_public_api_ts.js`, it only contains `occ` codes. `components and core` both are in `main.js`.